geomstats/geometry/pre_shape.py

```diff
@@ -40,10 +41,12 @@ class PreShapeSpace(EmbeddedManifold):
     """
 
     def __init__(self, k_landmarks, m_ambient):
+        embedding_manifold = Matrices(k_landmarks, m_ambient)
         super(PreShapeSpace, self).__init__(
```

Comment:
Is the constructor of FiberBundle called here?

Reply:
`super(..).__init__` calls all the constructors of the parent classes